bundles/org.eclipse.ui.editors/src/org/eclipse/ui/internal/texteditor/stickyscroll/StickyScrollingControl.java

```diff
@@ -77,6 +77,12 @@
  */
 public class StickyScrollingControl {
 
+	/**
+	 * The amount of visible source lines that should be shown as minimum. The sticky lines control
```

Comment:
you mean the amount of lines *in the editor*? Would be good to make this explicit in the java doc and manybe in the name of the constant

Reply:
Done